content/en/docs/Reference/build_raw_images_with_qemu.md

```diff
@@ -0,0 +1,117 @@
+---
+title: "Build Raw images with QEMU"
+linkTitle: "Build Raw images with QEMU"
+weight: 5
+description: This article shows how to bring your own image with Kairos, and build a Kairos derivative from scratch using base container images from popular distributions such as Ubuntu, Fedora, openSUSE, etc.
+---
+
+This page provides a reference guide on how to build raw images for Kairos using QEMU. It covers the process of using a default cloud configuration and a script to generate bootable images. The cloud configuration can be customized to suit different use cases.
```

Comment:
We need to explain the difference between this method and the alternative with auroraboot. When should one be used over the other? What does this offer that the other one doesn't (and vise versa).

Reply:
good point, addressed in https://github.com/kairos-io/kairos-docs/pull/355/commits/4b87dc7e2a3df4bab31fa73ab013d816618e55bd !